Description: Fixes in html doc which prevent correct man page generation
Author: Thibaut GRIDEL <tgridel@free.fr>

--- giflib-4.1.6.orig/doc/gif2x11.html
+++ giflib-4.1.6/doc/gif2x11.html
@@ -9,7 +9,7 @@ Go to <a href="index.html">index page</a
 
 <CENTER><H1>gif2x11</H1></CENTER>
 
-A program to display images saved as GIF files under X window system.<P?
+A program to display images saved as GIF files under X window system.<P>
 
 <H1>Usage:</H1>
 
--- giflib-4.1.6.orig/doc/text2gif.html
+++ giflib-4.1.6/doc/text2gif.html
@@ -50,7 +50,7 @@ Line.
 <DT> [-c R G B]
 <DD> The color to use as the foreground color.  White by default.<P>
 
-<DT> [-t "Text"]
+<DT> [-t &quot;Text&quot;]
 <DD> One line of text can be provided on the command
    line.  Note you must encapsulate the Text within quotes if it has spaces
    (The quotes themselves are not treated as part of the text).  If no -t
--- giflib-4.1.6.orig/doc/rgb2gif.html
+++ giflib-4.1.6/doc/rgb2gif.html
@@ -33,12 +33,12 @@ Screen.
 
 <DT> [-c #Colors] 
 <DD> Specifies number of colors to use, in bits per pixels, so
-   '-c 8' specifies actually 256 colors (maximum and default).<P>
+   `-c 8' specifies actually 256 colors (maximum and default).<P>
 
 <DT> [-1]
 <DD> Only one file in the format of RGBRGB... triplets (Each of R, G, B
    is a byte) is read from input.  This file size is 3 * Width * Height (see
-   '-s' below.  If stdin is used for input, this option is implicitly applied.
+   `-s' below.  If stdin is used for input, this option is implicitly applied.
    The default (if not '-1') is 3 files with the names RGBFile.R, RGBFile.G,
    RGBFile.B, each of which is Width * Height bytes.<P>
 
